root/report/ReportsIndex.js

```diff
@@ -449,12 +449,12 @@ const ReportsIndex = ({$c}: {$c: CatalystContextT}) => (
                 to VA`)}
           </a>
         </li>
-        <li>
+        {/*<li>
           <a href="/report/RecordingsSameNameDifferentArtistsSameName">
             {l(`Recordings with the same name by different artists
                 with the same name`)}
           </a>
-        </li>
+        </li>*/}
```

Comment:
This will likely drop all translations for this message. Could it be moved to an unused `const` instead?

Reply:
I could just set display: none on the link too, I guess